coral-spark/src/main/java/com/linkedin/coral/spark/IRRelToSparkRelTransformer.java

```diff
@@ -315,7 +315,7 @@ private Optional<RexNode> convertFuzzyUnionGenericProject(RexCall call) {
       if (call.getOperator() instanceof GenericProjectFunction) {
         RelDataType expectedRelDataType = call.getType();
         String expectedRelDataTypeString =
-            RelDataTypeToSparkDataTypeStringConverter.convertRelDataType(expectedRelDataType);
+            RelDataTypeToHiveTypeStringConverter.buildHiveTypeString(expectedRelDataType);
```

Comment:
I think we may keep using `convertRelDataType` as the method name given the class name is `RelDataTypeToHiveTypeStringConverter`.